assets/js/modules/ads/components/setup/SetupFormPAX.js

```diff
@@ -82,9 +94,22 @@ export default function SetupFormPAX( {
 				<ConversionIDTextField hideHeading />
 			</div>
 
+			{ isDuplicateAdsIDDetected && (
+				<WarningNotice className="googlesitekit-ads-setup__ads-id-conflict-warning">
+					{ __(
+						"This Conversion ID is already in use via Google for WooCommerce plugin. We don't recommend adding it in Site Kit, as it may result in inaccurate measurement for your Ads campaign conversions.",
```

Comment:
```suggestion
						"This Conversion ID is already in use via Google for WooCommerce plugin. We don’t recommend adding it in Site Kit, as it may result in inaccurate measurement for your Ads campaign conversions.",
```